style.css

```diff
@@ -0,0 +1,844 @@
+.my-project {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto auto;
+  gap: 10px;
+  padding: 10px;
+}
+
+h1 {
+  position: absolute;
+  width: 181px;
+  height: 50px;
+  left: 16px;
+  top: 20px;
+  font-family: sans-serif;
+  font-style: normal;
```

Comment:
Please notice that the fonts used in Figma are the following:
For headers
![image](https://user-images.githubusercontent.com/42799579/198783688-792272d8-88eb-42d8-9c9b-fe5305c99c55.png)

For plain text:
![image](https://user-images.githubusercontent.com/42799579/198783773-c9693663-3fd3-42d2-a853-5d49a2acd78e.png)

Kindly adjust your fonts to make your project look as similar as possible to the design.

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.